spec.html

```diff
@@ -19219,7 +19219,7 @@ <h1>Runtime Semantics: Evaluation</h1>
               1. If _argList_ has no elements, return *undefined*.
               1. Let _evalArg_ be the first element of _argList_.
               1. If the source text matched by this |CallExpression| is strict mode code, let _strictCaller_ be *true*. Otherwise let _strictCaller_ be *false*.
-              1. [id="step-callexpression-evaluation-direct-eval"] Return ? PerformEval(_evalArg_, _strictCaller_, *true*).
+              1. [id="step-callexpression-evaluation-direct-eval"] Return ? PerformEval(_evalArg_, |Script|, _strictCaller_, *true*).
```

Comment:
```suggestion
              1. [id="step-callexpression-evaluation-direct-eval"] Return ? PerformEval(_evalArg_, _strictCaller_, *true*).
```

There's no need to pass `|Script|` to PerformEval; PerformEval is 'hard-coded' to use `|Script|`.

Reply:
Good point, that makes this PR even smaller.